AntistasiOfficial.Altis/establishCamp.sqf

```diff
@@ -50,25 +50,27 @@ _pos = position _road findEmptyPosition [1,30,guer_veh_truck];
 
 _vehicle=[_pos, 0,guer_veh_truck, side_blue] call bis_fnc_spawnvehicle;
 _camion = _vehicle select 0;
-_grupo = _vehicle select 2;
+_camion deleteVehicleCrew driver _camion;
```

Comment:
Never looked at this code before, but generally i don't like the idea to create AI manned vehicle with `bis_fnc_spawnvehicle` to later remove that (those) AI with `deleteVehicleCrew`. Much more optimal would be to spawn empty vehicle and then just man it. Also in that particular case only driver is removed. If (for whatever reason) vehicle spawns multicrewed, it would require additional removals.

Reply:
Show me how ;-)

Reply:
My code is the quick fix, do the best ;)